### ISSUE TITLE
Support GitLab

### ISSUE THREAD

**Issue by vincent-fuchs** [societe-generale/github-crawler#82]
would be nice to have similar support, but for GitLab instead of GitHub !

**Comment by vincent-fuchs** [comment]
initiating something in https://github.com/societe-generale/github-crawler/tree/gitLabImpl

**Comment by Lucas-C** [comment]
Is this implemented ? _cf._ https://github.com/societe-generale/github-crawler#gitlab-support

**Comment by vincent-fuchs** [comment]
> Is this implemented ? _cf._ https://github.com/societe-generale/github-crawler#gitlab-support

yes, it is. Issue is not closed because there's not full support, but I think the current support is enough for most use cases. try it and let us know !
